games-emulation/bsnes-jg/bsnes-jg-2.0.0.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DOCS_BUILDER="doxygen"
+DOCS_DIR="objs/doc"
+
+inherit docs toolchain-funcs
+
+MY_PN=${PN%-*}
+MY_P=${MY_PN}-${PV}
+DESCRIPTION="Jolly Good Fork of bsnes"
+HOMEPAGE="https://gitlab.com/jgemu/bsnes"
+if [[ "${PV}" == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.com/jgemu/${MY_PN}.git"
+else
+	SRC_URI="https://gitlab.com/jgemu/${MY_PN}/-/archive/${PV}/${MY_P}.tar.bz2"
+	S="${WORKDIR}/${MY_P}"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+fi
+
+LICENSE="
+	ISC GPL-3+ LGPL-2.1+ MIT ZLIB
+	examples? ( 0BSD )
+"
+SLOT="1"
+IUSE="examples +jgmodule shared"
+REQUIRED_USE="
+	|| ( examples jgmodule shared )
```

Comment:
Eh, I guess it compiles/installs nothing if none of these are set? :)

Reply:
Yep, the Makefile would print `make: Nothing to be done for 'all'.` in that case.